### ISSUE TITLE
Allow resume of process but be able to define the step to use next

### ISSUE THREAD

**Issue by caperaven** [caperaven/crs-process-api#2]
Currently it just resumes and continues to next_step.
There are scenarios where you want to resume but from a particular step.

**Comment by caperaven** [comment]
This is an example of a pause step.

`            
pause: {
    type: "system",
    action: "pause",
    next_step: "close"
},
`

Normally to execute the resume you would bind the button as

`<button click.call="resume">Close</button>`

You can now bind it so that the resume will pass on the next step as a parmater.

<button click.call="resume('log')">Log and Close</button>

If you don't provide the parameter it will execute the normal next step as defined on the pause step.
In other words the "close" step.
If you do provide a next step, it will instead call the defined step name.
In this case the "log" step.

crs-process-api@0.0.42 +
